src/plugins/intel_cpu/src/emitters/x64/jit_load_store_emitters.cpp

```diff
@@ -234,11 +234,13 @@ void jit_load_emitter::load_bytes(const Vmm &vmm, const Xbyak::Reg64 &reg, int o
             has_xmm_block = true;
         }
 
+        if (!one_of(bytes_to_load, 0, 1, 2, 3, 4, 8, 16)) {
```

Comment:
That would be awesome if you can write a comment why we need `vpxor` here and why it is inserted only for some byte sizes.

Reply:
@dmitry-gorokhov the explanation comment is added.